.ci/setup_env_ubuntu.sh

```diff
@@ -42,6 +42,10 @@ declare -A packages=( \
 	[redis]="redis-server" \
 )
 
+if [ "${NAME}" == "Ubuntu" ] && [ "$(echo "${VERSION_ID} < 20.04" | bc -q)" == "1" ]; then
+	packages[cri-containerd_dependencies]+=" btrfs-tools"
```

Comment:
Does btrfs-tools have an alternative in 20.04?

Reply:
@cmaf I do not think so